pilot/prompts/development/update_plan.prompt

```diff
@@ -3,14 +3,23 @@ You are working on an {{ app_type }} called "{{ name }}". To give you a better i
 {{ app_summary }}
 ```
 
-Development plan for that {{ task_source }} was created and the {{ task_source }} was then broken down to smaller tasks so that it's easier for development. Here is development plan broken down in tasks:
-```{% for task in development_tasks %}
+Development plan for that {{ task_source }} was created and the {{ task_source }} was then broken down to smaller tasks so that it's easier for development.
+
+Here are tasks that are finished so far:
+```{% for task in finished_tasks %}
 - Task #{{ loop.index }}
-Description: {{ task['description'] }}
-Finished: {{ task['finished'] }}
+Description: {{ task }}
+
 {% endfor %}```
 
-As you can see in the list above, tasks that have flag "finished == True" are already finished and tasks with flag "finished == False" still have to be done. Last task you were working on has this description:
+Here are tasks that still have to be implemented:
+```{% for task in not_finished_tasks %}
+- Task #{{ loop.index }}
```

Comment:
This will show task numbers from 1 again, do we want to start the numbering from len(finished) + loop.index instead?